app/src/main/res/layout/content_main.xml

```diff
@@ -54,22 +54,16 @@
             android:gravity="center"
             android:foregroundGravity="center"
             android:visibility="invisible"
-            ></TextView>
+            >
+        </TextView>
 
-        <androidx.swiperefreshlayout.widget.SwipeRefreshLayout
```

Comment:
is there any reason to remove this SwipeRefreshLayout??

Reply:
Yes. It was not implemented properly. Because of that sort and filter buttons in Mainactivity became invisible. 
Also, I thought as the data we are displaying is not going to change in every few seconds and a new message is anyway going to come as a push notification, there is no point in having SwipeRefreshLayout. So I removed it.